cypress/integration/test/members.spec.js

```diff
@@ -0,0 +1,86 @@
+const newMember = {
+    name: 'name',
+    facebookUrl: 'facebookUrl',
+    instagramUrl: 'instagramUrl',
+    linkedinUrl: 'linkedinUrl',
+    image: 'image',
+    description: 'description'
+}
+const urlEndpoint = "http://localhost:3000/members/"
+const id = 22;
+
+
+describe("MEMBERS TESTING", () => {
+
+    it("Get all members", () => {
+
+        cy.request('GET', urlEndpoint)
+            .should((response) => {
+                expect('Content-Type', /json/)
+                expect(response.status).to.eq(200)
+                expect(response.body).to.not.be.null
```

Comment:
Eze esta muy bien en general. Cambiaria el `expect(X).to.not.be.null` por un `expext(X).to.deep.equal(X)` es decir, no testear por lo que no es, sino por lo que se espera.

Reply:
Hola Maxi! Muchas gracias por el feedback, testing me cuesta un poco. Ya corregí el método de cypress y aparentemente quedó bien, te adjunto una captura. 

![4](https://user-images.githubusercontent.com/79542271/165318938-da33ddcf-0ff0-49d5-bc42-9e58b72a8ade.JPG)